tools/clang/test/CodeGenHLSL/quick-test/rawbufferloadstore_64bit_6_2.hlsl

```diff
@@ -0,0 +1,104 @@
+// // RUN: %dxc -E main -T cs_6_2 %s | FileCheck %s
+
+struct TestData { 
+  int64_t3 v3;
+  int64_t4 v4;
+};
+
+ByteAddressBuffer srv0 : register(t0); 
+RWByteAddressBuffer uav0 : register(u0); 
+
+StructuredBuffer<TestData> srv1 : register(t1);
+RWStructuredBuffer<TestData> uav1 : register(u1);
+
+[numthreads(1, 1, 1)]
+void main(uint GI : SV_GroupIndex) {
+
+  int64_t3 vec3 = srv0.Load<int64_t3>(0);
+// CHECK: call %dx.types.ResRet.i32 @dx.op.rawBufferLoad.i32(i32 139, %dx.types.Handle %srv0_texture_rawbuf, i32 0, i32 undef, i8 15, i32 8)
+// CHECK: extractvalue %dx.types.ResRet.i32 %{{[a-zA-Z0-9]+}}, 0
```

Comment:
Instead of deciding which registers get names or not, I typically use {{.*}}. {{\w+}} would be ideal but I'm not sure the regex flavor here supports it.